src/components/Buildbaron/BuildBaron.test.tsx

```diff
@@ -0,0 +1,263 @@
+            assigneeDisplayName: null,
+            resolutionName: "Declined",
+            created: "2020-09-23T15:31:33.000+0000",
+            updated: "2020-09-23T15:33:02.000+0000",
+            status: {
+              id: "5",
+              name: "Resolved",
+            },
+          },
+        },
+        {
+          key: "EVG-12346",
+          fields: {
+            summary: "This is a random Jira ticket title 2",
+            assigneeDisplayName: "Some Name",
+            resolutionName: "Declined",
+            created: "2020-09-18T16:58:32.000+0000",
+            updated: "2020-09-18T19:56:42.000+0000",
+            status: {
+              id: "6",
+              name: "Closed",
+            },
+          },
+        },
+        {
+          key: "EVG-12347",
+          fields: {
+            summary: "This is a random Jira ticket title 3",
+            assigneeDisplayName: "Backlog - Evergreen Team",
+            resolutionName: "Declined",
+            created: "2020-09-18T17:04:06.000+0000",
+            updated: "2020-09-18T19:56:29.000+0000",
+            status: {
+              id: "1",
+              name: "Open",
+            },
+          },
+        },
+      ],
+      search:
+        '(project in (EVG)) and ( text~"docker\\\\-cleanup" ) order by updatedDate desc',
+      source: "JIRA",
+      featuresURL: "",
+    },
+  },
+};
+
+const mocks = [
+  {
+    request: {
+      query: GET_BUILD_BARON,
+      variables: {
+        taskId,
+        execution,
+      },
+    },
+    result: {
+      data: buildBaronQuery,
+    },
+  },
+  {
+    request: {
+      query: FILE_JIRA_TICKET,
+      variables: { taskId },
+    },
+    result: {
+      data: {
+        bbCreateTicket: true,
+      },
+    },
+  },
+  {
+    request: {
+      query: GET_CREATED_TICKETS,
+      variables: {
+        taskId,
+      },
+    },
+    result: {
+      data: {
+        bbGetCreatedTickets: [],
+      },
+    },
+  },
+  {
+    request: {
+      query: GET_SPRUCE_CONFIG,
+    },
+    result: {
+      data: {
+        spruceConfig: {
+          bannerTheme: "information",
+          banner: "",
+          ui: {
+            userVoice: "",
+          },
+          jira: {
+            host: "jira.mongodb.org",
+          },
+        },
+      },
+    },
+  },
+  {
+    request: {
+      query: GET_USER,
+    },
+    result: {
+      data: {
+        userId: "mohamed.khelif",
+        displayName: "Mohamed Khelif",
+      },
+    },
+  },
+  {
+    request: {
+      query: GET_TASK_EVENT_DATA,
+      variables: {
+        taskId,
+      },
+    },
+    result: {
+      data: {
+        task: {
+          __typename: "Task",
+          id: taskId,
+          status: "started",
+          failedTestCount: 0,
+        },
+      },
+    },
+  },
+  {
+    request: {
+      query: GET_USER,
+    },
+    result: {
+      data: {
+        userId: "mohamed.khelif",
+        displayName: "Mohamed Khelif",
+      },
+    },
```

Comment:
I think this request was mocked twice (on line 137 also), Is this intentional?

Reply:
removed it, thanks!